applications/GeoMechanicsApplication/tests/line_interface_elements/Dirichlet_multi_stage/ProjectParameters_stage1.json

```diff
@@ -0,0 +1,147 @@
+{
+    "problem_data": {
+        "problem_name":         "Dirichlet_multi_stage",
+        "start_time":           0.0,
+        "end_time":             2.0,
+        "echo_level":           1,
+        "parallel_type":        "OpenMP",
+        "number_of_threads":    1
+    },
+    "solver_settings": {
+        "solver_type":                        "U_Pw",
+        "model_part_name":                    "PorousDomain",
+        "domain_size":                        2,
+        "model_import_settings":              {
+            "input_type":       "mdpa",
+            "input_filename":   "3+3_line"
+        },
+        "material_import_settings":              {
+            "materials_filename":       "../common/MaterialParameters.json"
+        },
+        "time_stepping":              {
+            "time_step":            1.0
+        },
+        "buffer_size":                        2,
+        "echo_level":                         1,
+        "clear_storage":                      false,
+        "compute_reactions":                  true,
```

Comment:
I think we could put this to false. I did some clean-up in the single stage tests (which is of course based of the same example), maybe you could diff and see which settings could be the same? The same holds then of course for the other jsons

Reply:
I have compared the single stage analysis files with those of the multi-stage analyses. For consistency, I have changed some flags for the multi-stage analyses which had no effect on the obtained results. Thanks for pointing this out.